pkg/services/libraryelements/database.go

```diff
@@ -260,6 +271,16 @@ func (l *LibraryElementService) getLibraryElement(c *models.ReqContext, uid stri
 	return dto, nil
 }
 
+// getLibraryElementByUid gets a Library Element by uid.
+func (l *LibraryElementService) getLibraryElementByUid(c *models.ReqContext) (LibraryElementDTO, error) {
+	return l.getLibraryElement(c, map[string]interface{}{"org_id": c.SignedInUser.OrgId, "uid": c.Params(":uid")})
+}
+
+// getLibraryElementByName gets a Library Element by name.
+func (l *LibraryElementService) getLibraryElementByName(c *models.ReqContext) (LibraryElementDTO, error) {
+	return l.getLibraryElement(c, map[string]interface{}{"name": c.Params(":name")})
```

Comment:
The name should be unique within an Org so I think you need org_id as a param here too?

Reply:
Ah, you're right, will change

Reply:
Also, I think we need `kind`, and `folder_id` here too